packages/block-library/src/site-logo/edit.js

```diff
@@ -250,6 +264,14 @@ const SiteLogo = ( {
 			</ResizableBox>
 		);
 
+	const syncSiteIconHelpText = createInterpolateElement(
+		__(
+			"Site Icons are what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. If you don't have one, you can set your logo to also be your icon. If you do have a custom site icon, you can upload that from the Site Icon settings in the <a>Customizer</a>!"
+		),
+		// eslint-disable-next-line jsx-a11y/anchor-has-content
+		{ a: <a href={ siteUrl + '/wp-admin/customize.php' } target="blank" /> }
```

Comment:
The URL could also be `/wp-admin/customize.php?autofocus[section]=title_tagline`, which opens up the Site Identity panel straight away.

Reply:
Brilliant, this makes it _much_ nicer!